Ext4Pkg/Ext4Pkg.dsc

```diff
@@ -20,11 +20,11 @@
   BUILD_TARGETS                  = DEBUG|RELEASE|NOOPT
   SKUID_IDENTIFIER               = DEFAULT
 
-!include MdePkg/MdeLibs.dsc.inc
-
 [BuildOptions]
   *_*_*_CC_FLAGS                       = -D DISABLE_NEW_DEPRECATED_INTERFACES
 
+!include MdePkg/MdeLibs.dsc.inc

```

Comment:
Do we really need this change now?

Reply:
It is reverting https://github.com/acidanthera/audk/commit/9a1136d880aeee16709d9e48309c52bc2829ddd8#diff-f97bcbb084b537538f503ed16bee9c75a91adb663c18aef2bddf3910138a9d24L23